WifiART/src/userinterface/ArtMvcView.java

```diff
@@ -181,6 +246,133 @@ private void setupOutputFileChooserPanel(JPanel panel) {
+        latitudePanel.add(latitudeFieldLabel);
+        latitudePanel.add(latitudeField);
+        latitudePanel.add(latitudeUncertaintyFieldLabel);
+        latitudePanel.add(latitudeUncertaintyField);
+        panel.add(latitudePanel);
+
+        JPanel longitudePanel = new JPanel();
+        longitudePanel.setLayout(new BoxLayout(longitudePanel, BoxLayout.X_AXIS));
+        longitudePanel.add(longitudeFieldLabel);
+        longitudePanel.add(longitudeField);
+        longitudePanel.add(longitudeUncertaintyFieldLabel);
+        longitudePanel.add(longitudeUncertaintyField);
+        panel.add(longitudePanel);
+
+        JPanel altitudePanel = new JPanel();
+        altitudePanel.setLayout(new BoxLayout(altitudePanel, BoxLayout.X_AXIS));
+        altitudePanel.add(altitudeFieldLabel);
+        altitudePanel.add(altitudeField);
+        altitudePanel.add(altitudeUncertaintyFieldLabel);
+        altitudePanel.add(altitudeUncertaintyField);
+        panel.add(altitudePanel);
+
+        JPanel altitudeTypeAndDatumPanel = new JPanel();
+        altitudeTypeAndDatumPanel.setLayout(new BoxLayout(altitudeTypeAndDatumPanel, BoxLayout.X_AXIS));
+        altitudeTypeAndDatumPanel.add(altitudeTypeComboBoxLabel);
+        altitudeTypeAndDatumPanel.add(altitudeTypeComboBox);
+        altitudeTypeAndDatumPanel.add(Box.createHorizontalGlue());
+        altitudeTypeAndDatumPanel.add(mapDatumComboBoxLabel);
+        altitudeTypeAndDatumPanel.add(mapDatumComboBox);
+        panel.add(altitudeTypeAndDatumPanel);
+
+        JPanel lciReservedFieldsPanel = new JPanel();
+        setupLciReservedFieldsPanel(lciReservedFieldsPanel);
+        panel.add(lciReservedFieldsPanel);
+
+    }
+    private void setupLciReservedFieldsPanel(JPanel lciReservedFieldsPanel) {
+        lciReservedFieldsPanel.setLayout(new BoxLayout(lciReservedFieldsPanel, BoxLayout.Y_AXIS));
+        lciReservedFieldsPanel.setBorder(BorderFactory.createTitledBorder(BorderFactory.createLineBorder(Color.BLACK), LCI_RESERVED_FIELDS_LABEL));
+        JPanel regLocAgreementCheckboxPanel = new JPanel();
+        regLocAgreementCheckboxPanel.setLayout(new FlowLayout(FlowLayout.LEFT));
+        regLocAgreementCheckboxPanel.add(regLocAgreementCheckbox);
+        lciReservedFieldsPanel.add(regLocAgreementCheckboxPanel);
+        JPanel regLocDseCheckboxPanel = new JPanel();
+        regLocDseCheckboxPanel.setLayout(new FlowLayout(FlowLayout.LEFT));
+        regLocDseCheckboxPanel.add(regLocDseCheckbox);
+        lciReservedFieldsPanel.add(regLocDseCheckboxPanel);
+        JPanel dependentStaCheckboxPanel = new JPanel();
+        dependentStaCheckboxPanel.setLayout(new FlowLayout(FlowLayout.LEFT));
+        dependentStaCheckboxPanel.add(dependentStaCheckbox);
+        lciReservedFieldsPanel.add(dependentStaCheckboxPanel);
+    }
+
+    private void setupZPanel(JPanel panel) {
+        panel.setLayout(new BoxLayout(panel, BoxLayout.Y_AXIS));
+
+        //Panel containing the title for this tab.
```

Comment:
nit: need an extra space after //. Here and below.

Reply:
Thanks, that seems to be a bad habit of mine.